insights/tests/parsers/test_saphostctrl.py

```diff
@@ -142,11 +143,11 @@ def test_saphostctrl_docs():
 
 def test_saphostctrl():
     sap = SAPHostCtrlInstances(context_wrap(SAPHOSTCTRL_HOSTINSTANCES_GOOD))
-    assert len(sap) == 10
-    assert sap.data[-2]['SapVersionInfo'] == '749, patch 401, changelist 1806777'
```

Comment:
@xiangce it might be good to leave this test in too for `data` since you added it as a property to maintain backward compatibility.

Reply:
Fixed.

And fixed the test data in rules per the parser. 

Thanks for checking!

Reply:
@bfahr - The items you pointed out were fixed, I went ahead and merged this PR. For further issues, please let me know.